.gitignore

```diff
@@ -26,6 +26,3 @@ _testmain.go
 # tools and virtualenv
 .idea
 env/
-
```

Comment:
do you want to version the binary?

Reply:
sure, I will try

Reply:
why?

Reply:
let's just add again the gitignore, preventing the binary from going to git

Reply:
👍